requirements/env_climada.yml

```diff
@@ -4,15 +4,15 @@ channels:
   - defaults
 dependencies:
   - bottleneck>=1.3
-  - cartopy>=0.20.0,<0.20.3  # 0.20.3 has an issue with geographic crs in plots
+  - cartopy>=0.20.0  # 0.20.3 has an issue with geographic crs in plots
   - cfgrib>=0.9.7,<0.9.10  # 0.9.10 cannot read the icon_grib files from https://opendata.dwd.de
   - contextily>=1.0
   - dask>=2.25
   - gdal!=3.4.1  # has an issue with fiona on Windows (#403)
   - geopandas>=0.8
   - h5py>=2.10
   - haversine>=2.3
-  - matplotlib>=3.2,<3.6  # 3.6 requires cartopy 0.21 (s. https://github.com/SciTools/cartopy/pull/2054)
+  - matplotlib==3.7 
```

Comment:
I think pinning a version is dangerous because we might get incompatible environments more quickly. What is the reason to pin to matplotlib v3.7?

Reply:
I plan to set minimal version numbers for all dependencies according to the currently installed versions (union between Windows, Linux and MacOS) for the develop branch. Hence the env_climada.yml is bound to change!

For the releases (and for the Conda spec) I'm going to to completely fix the version numbers of all dependencies (but not the builds). The idea is to help with reproducibility and in the mean time avoid trouble shooting conda installations.
This will not succeed completely because the indirect dependencies still have some flexibility, but at least it's a start.

In setup.py on the other hand, I don't plan to change anything, because - if someone is able to install climada based on pip alone, they must be mighty savvy and probably have their own agenda. (E.g. Euler)